src/components/ui/Search.jsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+import { useState } from "react";
+
+export default function Search() {
+  const [searchTerm, setSearchTerm] = useState('');
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    window.location.href = `/article?search=${searchTerm}`;
+  }
+  const handleInputChange = (event) => {
+    setSearchTerm(event.target.value);
+  };
+  return (
+    <div className="l-header__nav__search">
+      <form onSubmit={handleSubmit}>
+        <button
+          type="submit"
+          aria-label="検索"
+          className="l-header__nav__searchButton"
+        ></button>
+        <label>
+          <input
+            type="text"
+            placeholder="キーワードを入力"
+            className="l-header__nav__searchText"
+            value={searchTerm}
+            onChange={handleInputChange}
```

Comment:
@VortexExpansion 
Pls consider debouncing

Reply:
@yabe-diverta san

The search here is not dynamic on every key stroke but are fetched only when the user presses the submit button.
Only the searchTerm is updated on every keystroke but the value is fetched only once after pressing submit so I think debounce here would not be a significant performance boost. 

Do you think implementing debounce here is still important ? 🙇

Reply:
Ah understood.
No need to implement it, thank you for clarification 🙇